engine/src/main/java/com/datatorrent/stram/plan/physical/PhysicalPlan.java

```diff
@@ -357,25 +417,51 @@ public PhysicalPlan(LogicalPlan dag, PlanContext ctx) {
           if (!container.operators.isEmpty()) {
             LOG.warn("Operator {} shares container without locality contraint due to insufficient resources.", oper);
           }
+          // TODO: Check if PT Operators conflict in anti-affinity, Pick the first container that does not conflict
```

Comment:
This check cannot be done in logical Plan validation. Since, 2 operators could be assigned to same container even if there is not locality constraint. This may occur if maxContainer value is set less than the needed number of containers for operator.